markdown/events/2021/bytesize-1-nf-core-into.md

```diff
@@ -33,3 +33,152 @@ The talk will be presented on Zoom and live-streamed on YouTube:
+
+We are also looking for ideas for what people should work on different tasks and all of this is being organized through Slack.
+
+So hop onto the new hackathon March 2021 Slack channel and either say that you would like to be involved or suggest any ideas. I should have put the link in here but there is also a webpage on the nf-core website for this.
+
+Maybe I can share my screen window quickly just so you can see what I am talking about. Here you can see I am on the <b>nf-core</b> website. If I hop onto the 'Events' page <https://nf-co.re/events>, you can see that we have a page about today's talk. This one right now with links and everything.
+
+There is also a page about the upcoming hackathon so if you click here, you will find the link to register. This is non-binding it does not cost anything to join a hackathon; it just helps us know roughly who is going to be there; also which projects people are interested in/working on, and if you want any of that cool stash then you have to put your address in so we can post it to you.
+
+And then there are details about when it is going to be and how it is going to work. And this page will grow as the hackathon gets closer.  You can also see some of the ideas that we are working on for different kinds of tasks to work on the project.
+
+Great, that is my 15 minutes! So as I finish every talk, come and join our community if you are
+not already. we operate on Github, Slack, Twitter, Youtube and you can find all the details on how to join all of those on our website.
+
+I did not introduce myself very much at the start but my name is Phil. I work at SciLifelab and I live in Sweden as well at the NGI. This lovely view I took with my new toy - it is about 70 meters above where I am sitting right now, when the sun was shining a couple of days ago
+
+With that I am happy to take any questions. Please pop them into the Zoom chat or Slack, and Renuka can relay to me.
+
+### Q&A
+Renuka:
+Thanks very much Phil and thanks everyone for listening. We have a question in the chat. Vidya wants to know whether there will be any talks on how to run <b>nf-core</b> on a grid system, like running on Slurm.
+
+Phil:
+Yeah I mean that is quite a specific question. Not so much nf-core, but that is more of a Nextflow issue we are going to talk about. One of the first talks will be about how to set up core configs.
+So we have a few different ways to bring in Nextflow configuration into the <b>nf-core</b> pipelines and to make that easy for you.
+
+We have plenty of people using nf-core pipelines on Slurm, myself included and so that would be a good place to start. There is a talk coming up about that and also head over to Slack, we have a channel called configs which is designed for questions like this.
```

Comment:
```suggestion
We have plenty of people using <b>nf-core</b> pipelines on Slurm, myself included and so that would be a good place to start. There is a talk coming up about that and also head over to Slack, we have a channel called configs which is designed for questions like this.
```